pages/common/hub.md

```diff
@@ -4,11 +4,7 @@
 > If set up as instructed by `hub alias`, one can use `git` to run `hub` commands.
 > More information: <https://hub.github.com>.
 
-- Clone a repository you own, using just the repository name rather than the full URL:
-
-`hub clone {{repo_name}}`
-
-- Clone another user's repository, using their GitHub username and the repository name:
+- Clone a repository, using its GitHub owner (can be omited if you own it) and the repository name:
```

Comment:
How about ?
```suggestion
- Clone a repository using its slug (owners can omit the username):
```
We recently used `slug` in a similar page: [git-fork](https://github.com/tldr-pages/tldr/blob/main/pages/common/git-fork.md)

Reply:
I don't know this word (I am not a native English speaker :innocent:). Is is slang? Could not find the meaning in [wordreference](https://www.wordreference.com/definition/slug). :thinking: 
Ah, OK, I found this comment: https://github.com/tldr-pages/tldr/pull/6013#discussion_r637399196.
Personally, I prefer using simple words that Atlassian jargon (is it?).

Reply:
It's not Atlassian-specific — it's a common word used for this type of thing (`user/repo`) 🤷🏻

Reply:
I'm also not totally convinced by the word itself, but the example is quite self-exlanatory and this way the user even learns a new word.

Reply:
OK, I am convinced now.

I have found the word `slug` in various tech documentations:
* BitBucket: https://support.atlassian.com/bitbucket-cloud/docs/what-is-a-slug/
* Azure DevOps: https://docs.microsoft.com/en-us/appcenter/distribution/vsts-deploy
* GitLab: https://docs.gitlab.com/ee/user/project/working_with_projects.html

etc.

Sorry but the initial "here is the Atlassian link" did not "sell" me the word. 😉

Reply:
Thank you @mfrw, @bl-ue and @marchersimon! I did the change in `fixup` commit c7726572 (in order to preserve the two commits where merging 🤓).

Reply:
@nicokosi we generally do a `squash & merge` :)

Reply:
I know but:
> If instead the PR author took the time to craft individual, informative messages for each commit, then use the Rebase and merge method, to honor that work and preserve the history of the changes.

See https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md#ii-handling-prs :innocent:

Reply:
I took that time. :nerd_face: